assets/src/edit-story/components/panels/design/shapeStyle/shapeStyle.js

```diff
@@ -54,12 +53,11 @@ function ShapeStylePanel({ selectedElements, pushUpdate }) {
       <Row>
         <Color
           ref={colorInputRef}
-          hasGradient
+          allowsGradient
           value={backgroundColor}
           isMultiple={backgroundColor === ''}
           onChange={onChange}
           label={__('Background color', 'web-stories')}
-          hasOpacity={!isBackground}
```

Comment:
Not seeing the opacity field anymore in the design panel for shapes, probably related to removal of this line?
![Screenshot 2021-08-03 at 16 49 59](https://user-images.githubusercontent.com/3294597/128027090-3713434c-e861-4ad4-b5af-28e3c4aa99ea.png)

Reply:
Oh! Well spotted - I wondered where that input went 😂